README.md

```diff
@@ -184,8 +184,7 @@ These optional properties configure a trust store with private CA certificates.
 
 | Environment Variable                | Default       | Description 
 | ------------------------------------|---------------|-------------
-| NHAIS_MONGO_DOCUMENTDB_TLS_ENABLED  | false         | (Optional) Enables TLS configuration for AWS DocumentDB
-| NHAIS_MONGO_TRUST_STORE_PATH        |               | (Optional) Path to trust store (used only in local/dev)
+| NHAIS_MONGO_TRUST_STORE_PATH        |               | (Optional) S3 path to trust store JKS
```

Comment:
Change to `NHAIS_MONGO_TRUST_STORE_URL`, Description '(Optional) URL of the trust store JWK. The only scheme currently supported is `s3`